diff --git a/views/shared/header/menu.jade b/views/shared/header/menu.jade
index 5b63d70..afa4be7 100644
--- a/views/shared/header/menu.jade
+++ b/views/shared/header/menu.jade
@@ -65,6 +65,9 @@ nav.toolbar(ng-controller='AuthCtrl', ng-class='{active: isToolbarHidden}')
                 a(target="_blank" href='http://habitrpg.wikia.com/wiki/Contributing_to_HabitRPG')=env.t('contributeToHRPG')
               li
                 a(target="_blank" href='http://habitrpg.wikia.com/wiki/')=env.t('overview')
+            ul.toolbar-submenu
+              li
+                a(href='/blackmarket/', target='_blank')='Black Market'
           ul.toolbar-controls
             li.toolbar-subscribe-button
               button(ng-if='!user.purchased.plan.customerId',ui-sref='options.settings.subscription',popover-trigger='mouseenter',popover-placement='bottom',popover-title=env.t('subscriptions'),popover=env.t('subDescription'),popover-append-to-body='true')=env.t('subscribe')
@@ -158,6 +161,9 @@ nav.toolbar(ng-controller='AuthCtrl', ng-class='{active: isToolbarHidden}')
               a(target="_blank" href='http://habitrpg.wikia.com/wiki/')=env.t('overview')
             li(ng-controller='SettingsCtrl')
               a(ng-click='showTour()', popover-placement='right', popover-trigger='mouseenter', popover=env.t('restartTour'))= env.t('showTour')
+      li.toolbar-button
+        a(href='/blackmarket/', target='_blank')
+          span='Black Market'
     ul.toolbar-subscribe(ng-if='!user.purchased.plan.customerId')
       li.toolbar-subscribe-button
         button(ui-sref='options.settings.subscription',popover-trigger='mouseenter',popover-placement='bottom',popover-title=env.t('subscriptions'),popover=env.t('subDescription'),popover-append-to-body='true')=env.t('subscribe')
